api/utils.py

```diff
@@ -0,0 +1,15 @@
+from datetime import datetime
```

Comment:
The fact that this file is called `utils.py` pretty much makes this PR a proof of that old saying:
> There are only two hard things in Computer Science: cache invalidation and naming things.
:P